pinot-core/src/test/java/com/linkedin/pinot/query/aggregation/AggregationMVGroupByMVQueriesTest.java

```diff
@@ -218,16 +218,16 @@ public void testAvgAggrMVGroupByMV() throws Exception {
     System.out.println(aggregationResult.getGroupByResult());
 
     Map<String, String> expectedGroupByResults = new HashMap<>();
-    expectedGroupByResults.put("[2147483647]", "2147483647.00000");
-    expectedGroupByResults.put("[536]", "536.00000");
-    expectedGroupByResults.put("[566]", "522.00000");
-    expectedGroupByResults.put("[532]", "519.75000");
-    expectedGroupByResults.put("[552]", "504.96429");
-    expectedGroupByResults.put("[529]", "499.44444");
-    expectedGroupByResults.put("[555]", "495.66667");
-    expectedGroupByResults.put("[570]", "495.00000");
-    expectedGroupByResults.put("[554]", "493.92857");
-    expectedGroupByResults.put("[545]", "493.03659");
+    expectedGroupByResults.put("[2147483647]", "2147483647.0");
+    expectedGroupByResults.put("[536]", "536.0");
+    expectedGroupByResults.put("[566]", "522.0");
+    expectedGroupByResults.put("[532]", "519.75");
+    expectedGroupByResults.put("[552]", "504.9642857143");
```

Comment:
I reverted the double format change because it's not related to this PR. If we want this change later, will file another PR.